### PR TITLE
Added support for FS NMU

### PR DESCRIPTION
**Pull request by jozefrebjak**
Please give a short description what your pull request is for

There was a problem for me to get this via YAML discovery, device also not responding to snmpwalks, so I used snmpget to get all mainly needed data for us. I hope it will be ok. 

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/polling/os/fs-nmu.inc.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?php
```
Comment (Jellyfrog) [inline includes/polling/os/fs-nmu.inc.php:1]: you should move this to the new format, see for example, https://github.com/librenms/librenms/pull/11932

Comment (jozefrebjak) [reply includes/polling/os/fs-nmu.inc.php:1]: @Jellyfrog to this function public `function discoverOS(): void` right ? I will move it

Comment (Jellyfrog) [reply includes/polling/os/fs-nmu.inc.php:1]: Yep yep

Comment (jozefrebjak) [reply includes/polling/os/fs-nmu.inc.php]: @Jellyfrog moved to new format

**Comment by cppmonkey** [comment]
@jozefrebjak  May want to have a look at my Tryin implementation.
image

Tracks a lot more details. Never submitted it, as it isn't very pretty. Hence tryin-devel was supposed to be a cleaned up version.

I have access to NMU running and a wide varaety of EDFA Pre-AMPs, Booster, OEO's and the like

- 1.02.26
- 1.03.15

https://github.com/cppmonkey/librenms/tree/tryin

**Comment by jozefrebjak** [comment]
@cppmonkey it’s looking good we can implement it. We bought the first one FS chassis with NMU and we have only A1,A2 with OEO for now.

**Comment by cppmonkey** [comment]
@jozefrebjak
The polling takes a great deal of time. I'd considered pre-caching the data before processing it.

The FS monitoring tool and SMT don't appear to use SNMP.
The early units used to be accessible via telnet and ssh. This allowed me to upgrade some to 1.02.26.

There are various changes between hardware and software revisions, impacting compatibility

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-67-release-changelog-august-2020/13285/1

**Pull request merged**